libnavui-status/src/main/java/com/mapbox/navigation/ui/status/model/StatusFactory.kt

```diff
@@ -0,0 +1,29 @@
+package com.mapbox.navigation.ui.status.model
+
+import androidx.annotation.DrawableRes
+import com.mapbox.navigation.base.ExperimentalMapboxNavigationAPI
+
+/**
+ * An object that creates new [Status] instances.
+ */
+@ExperimentalMapboxNavigationAPI
```

Comment:
We use `ExperimentalMapboxNavigationAPI` annotation to have an ability to add new parameters without default values, right?